cfme/tests/containers/test_breadcrumbs.py

```diff
@@ -16,74 +12,50 @@
 from cfme.containers.replicator import Replicator
 from cfme.containers.template import Template
 from cfme.containers.volume import Volume
-from cfme.containers.provider import ContainersProvider, navigate_and_get_rows
+from cfme.containers.provider import ContainersProvider
+from cfme.utils.appliance.implementations.ui import navigate_to
+from cfme.utils import testgen
 
 
 pytestmark = [
-    pytest.mark.usefixtures('setup_provider'),
+    pytest.mark.usefixtures('setup_provider_modscope'),
+    pytest.mark.usefixtures('has_persistent_volume'),
     pytest.mark.tier(1)]
-pytest_generate_tests = testgen.generate([ContainersProvider], scope='function')
-
-
-class DataSet(object):
-    def __init__(self, obj, obj_base_name):
-        self.obj = obj
-        self.obj_base_name = obj_base_name
-
-
-DATA_SETS = [
-    DataSet(Service, 'service'),
-    DataSet(Route, 'route'),
-    DataSet(Project, 'project'),
-    DataSet(Pod, 'pod'),
-    DataSet(Image, 'image'),
-    DataSet(ContainersProvider, 'provider'),
-    DataSet(ImageRegistry, 'image regist'),
-    DataSet(Node, 'node'),
-    DataSet(Replicator, 'replicator'),
-    DataSet(Template, 'template'),
-    DataSet(Volume, 'volume')
+pytest_generate_tests = testgen.generate([ContainersProvider], scope='module')
```

Comment:
Optional: please use mark.provider fixture instead. please take this into account and update tests when you fix them

example:
`
@pytest.mark.provider(
    [VMwareProvider, RHEVMProvider, OpenStackProvider, AzureProvider],
    scope="module"
)
`